index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
   </head>
```

Comment:
Required changes: 
- [ ] I would suggest we and make use of the [GitHub inline comments feature](/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/ADD_LINK_TO_VIDEO_OR_ARTCILE) at least once to leave review comments. Please make sure to leave the comment under a line or block of code that fits what the context the comment is about. This way the review comes off as complete and deliberately helpful. 
- [ ] Kindly make this change then submit for another re-review. Thank you!

Reply:
@2bleO  you've done well using Inline comments, but you should add at least 2, also please re-add the summary with it too so the review looks complete